deep_speech_2/decoders/swig/setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
change to `#! /usr/bin/env bash`, see https://stackoverflow.com/questions/16365130/the-difference-between-usr-bin-env-bash-and-usr-bin-bash

Reply:
Done